pootle/templates/user/includes/profile_teams.html

```diff
@@ -0,0 +1,14 @@
+{% if is_superuser %}
+<h3>Site administrator</h3>
```

Comment:
hmm - im not sure whether we want to keep it at all - but ill move it to py and add `_()`